src/django/all_neighborhoods.json

```diff
@@ -0,0 +1 @@
ile":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/yorba-linda-ca/yorba-linda-ca.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"aa441f1b-e65d-4463-9040-8927efa74cad"},{"uuid":"2195cafc-ab5e-4c03-93d7-8b3972fef063","createdAt":"2017-04-29T13:59:09.646200Z","modifiedAt":"2017-05-10T19:04:49.694252Z","name":"morristown-nj","label":"Morristown, NJ","state_abbrev":"NJ","boundary_file":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/morristown-nj/morristown-nj.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"db513545-6e29-43d1-b6d2-dce117571981"},{"uuid":"011fca4f-0b85-4408-a5b8-9d9c6729a419","createdAt":"2017-04-29T13:59:03.418664Z","modifiedAt":"2017-05-10T19:04:49.700683Z","name":"missoula-mt","label":"Missoula, MT","state_abbrev":"MT","boundary_file":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/missoula-mt/missoula-mt.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"0c6c69ad-a49c-49d5-9c62-e53f74946130"},{"uuid":"8b1de8f3-efd5-4a86-bab2-eb2e2c77d2be","createdAt":"2017-04-29T13:58:23.184349Z","modifiedAt":"2017-05-10T19:04:50.011095Z","name":"yuma-az","label":"Yuma, AZ","state_abbrev":"AZ","boundary_file":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/yuma-az/yuma-az.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"6c9145fa-f967-4161-8af2-b1985b0d9f07"},{"uuid":"1cfa1d33-7381-4042-9ac9-d6bcb888f8d3","createdAt":"2017-04-29T13:58:24.329914Z","modifiedAt":"2017-05-10T19:04:50.020047Z","name":"fayetteville-ar","label":"Fayetteville, AR","state_abbrev":"AR","boundary_file":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/fayetteville-ar/fayetteville-ar.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"b169a4c3-22b5-4a0c-89b0-90a8a36e1ac3"},{"uuid":"cc8eb9d3-e448-4289-89d8-9a8b6393bdf6","createdAt":"2017-05-02T14:26:21.469643Z","modifiedAt":"2017-05-17T14:23:02.916681Z","name":"raleigh-nc","label":"Raleigh, NC","state_abbrev":"NC","boundary_file":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/raleigh-city-nc/raleigh-city-nc.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"56e10fe5-1eed-4c56-8266-66d1a77c6360"},{"uuid":"88157f37-6932-4ac1-addd-fa5f4b747fa3","createdAt":"2017-04-29T13:58:24.585247Z","modifiedAt":"2017-05-10T19:04:50.028494Z","name":"bentonville-ar","label":"Bentonville, AR","state_abbrev":"AR","boundary_file":"https://production-pfb-storage-us-east-1.s3.amazonaws.com/neighborhood_boundaries/bentonville-ar/bentonville-ar.zip","visibility":"public","organization":"c3bb5ab9-b6d3-490b-9362-c6b59df56bbd","last_job":"552576de-45e5-484e-94bb-2d0f2b5fd36f"},{"uuid":"545d8b1d-
```

Comment:
Doh.  A rogue `git add` command.  Surprised it only pulled in that, since I have a variety of other scratch files laying around in the project directory...